controllers/sms-gateway.js

```diff
@@ -8,9 +8,10 @@ var async = require('async'),
     recordUtils = require('./record-utils'),
     STATUS_MAP = {
       UNSENT: 'received-by-gateway',
+      PENDING: 'forwarded-by-gateway',
```

Comment:
Do we really want a `PENDING` enum that means something different than the message state `pending`?